tests/monitoring_test.go

```diff
@@ -199,14 +376,16 @@ var _ = Describe("[Destructive] Monitoring Tests", func() {
 
 			By("Metric stays the same because we don't support this provisioner")
 			Consistently(func() int {
-				return getMetricValue(f, "kubevirt_cdi_incomplete_storageprofiles")
-			}, metricPollingTimeout, metricPollingInterval).Should(Equal(originalMetricVal))
+				return countMetricLabelValue(f, "kubevirt_cdi_storageprofile_info", "complete", "false")
+			}, metricConsistentPollingTimeout, metricPollingInterval).Should(BeZero())
```

Comment:
Same question about ditching originalMetricVal. How do we know the env didn't carry a positive value for this metric prior to the test

Reply:
I'll re-add it too.